docs/Secure-Coding-Guide-for-Python/CWE-682/CWE-191/README.md

```diff
@@ -1,3 +1,183 @@
 # CWE-191, Integer Underflow (Wrap or Wraparound)
 
-PLACEHOLDER for # CWE-1335: Promote readability and compatibility by using mathematical written code with arithmetic operations instead of bit-wise operations link
+Ensure that integer overflow is properly handled in order to avoid unexpected behavior. Python data types can be divided into two categories:
+
+- Built-in types such as `int`, `float`, or `complex` [[Python 2024]](https://docs.python.org/3.9/library/stdtypes.html). These types are provided by classes and are protected against overflows.
+
+- Primitive types such as `numpy`, `time`, or `ctype`  share issues known from `C` , or `C++`  and are often used to interact with the operating system, or for efficiency. Developers should follow the `C` guidelines for rules to follow.
```

Comment:
There is a word missing: "... interact with the operating system, or (?) for efficiency..."

Reply:
changed as discussed